docs/sources/helm-charts/mimir-distributed/run-production-environment-with-helm/_index.md

```diff
@@ -286,7 +286,7 @@ Your Grafana Mimir cluster can now ingest metrics in production.
 
 ## Configure clients to write metrics to Mimir
 
-To configure each client to remote-write metrics to Mimir, refer to [Configure Prometheus to write to Grafana Mimir]({{< relref "../get-started-helm-charts#configure-prometheus-to-write-to-grafana-mimir" >}})
+To configure each client to remote-write metrics to Mimir, refer to [Configure Prometheus to write to Grafana Mimir](https://grafana.com/docs/helm-charts/mimir-distributed/latest/get-started-helm-charts/gs-external-access/#configure-prometheus-to-write-to-grafana-mimir)
```

Comment:
Good call out! I believe I updated the URLs to use proper version substitution syntax per https://grafana.com/docs/writers-toolkit/write/shortcodes/#about-version-substitution, so that the URLs will always resolves to the current version.